smqtk_detection/impls/detect_image_objects/resnet_frcnn.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+
+import torch
+import torchvision.models as models
+from torchvision import transforms
+from torchvision.ops import boxes as box_ops
+import torch.nn.functional as F
+from torchvision.models.detection.roi_heads import RoIHeads
```

Comment:
These imports will need to be guarded by a try/except so that importing this sub-module does not fail when these "optional" dependencies are not present in the environment.